elephantdsp-roomreverb/loong.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/PKGBUILD b/PKGBUILD
+index 4b853a3..dca66b1 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
+@@ -78,7 +78,13 @@ prepare() {
+   patch -p1 -i "$srcdir/no-copy-plugin.patch"
+ 
+   # use system JUCE
+-  patch -p1 -i "$srcdir/devendor-juce.patch"
++  #patch -p1 -i "$srcdir/devendor-juce.patch"
++  # use userlib JUCE to modify platform detection
++  git submodule init Libs/JUCE
++  git config submodule.Libs/JUCE.url "$srcdir/github.com-JUCE"
++  git -c protocol.file.allow=always submodule update
++  dos2unix -o $srcdir/elephantdsp-roomreverb/Libs/JUCE/extras/Build/CMake/juce_runtime_arch_detection.cpp
++  patch -p1 -i "$srcdir/juce_platform.patch"
+ }
+ 
+ build() {
+@@ -130,3 +136,8 @@ package_elephantdsp-roomreverb-vst3() {
+   install -vd "$pkgdir/usr/lib/vst3"
+   cp -vr VST3/RoomReverb.vst3 "$pkgdir/usr/lib/vst3"
+ }
++
++source+=('github.com-JUCE::git+https://github.com/juce-framework/JUCE'
```

Comment:
Please fix on a certain commit to provide reproducible builds.

Reply:
Original PKGBUILD from ArchLinux all use the newest main branch.